src/package/nuspec/TestPlatform.CLI.nuspec

```diff
@@ -28,8 +28,8 @@
     <file src="licenses\LICENSE_NET.txt" target="" />
     <!-- Add a third party notice file -->
     <file src="ThirdPartyNotices.txt" target="" />
-    
+
     <!-- excluding dbghelp and cpp as they cannot be re-distributed with sdk, but need to be distributed with VS -->
-    <file src="$TargetFramework$\**\*.*" exclude="**\*.pdb;**\dbghelp.dll;**\*TestTools.CppUnitTestFramework*" target="contentFiles\any\$TargetFramework$" />
+    <file src="$TargetFramework$\**\*.*" exclude="**\*.pdb;**\*.xml;**\dbghelp.dll;**\*TestTools.CppUnitTestFramework*" target="contentFiles\any\$TargetFramework$" />
```

Comment:
Why are you excluding the xml files?

Reply:
the code analysis rule ide0005 in this repo was not enforced due to `GenerateDocumentationFile` set to `false` (see https://github.com/dotnet/roslyn/issues/41640#issuecomment-985780130). Enabling this property added enforcement of new code rules that relate to documentation annotations and generation of the output .xml doc files. If you think these files should be part of the package, I can remove the filter and update the assertion for the number of files in the nuget package accordingly.

Reply:
I would go for including them but let's see what @MarcoRossignoli and @nohwnd think.

Reply:
We should avoid to change the layout of the packages, it's sensible. As long as we ensure that the final layout didn't change I'm fine.

Reply:
As stated in the other comment, this xml exclusion only removes `Microsoft.TestPlatform.CLI.17.6.0-dev\contentFiles\any\netcoreapp3.1\Microsoft.TestPlatform.VsTestConsole.TranslationLayer.xml` from the package (without excluding xml files, the package layout would get extra xml files due to `GenerateDocumentationFile`).